### PR TITLE
Update the home page tests

### PR DESCRIPTION
**Pull request by DavidKohav**
#138 
In this pr i will update the behat test in needed and the phantomcss.

### CODE REVIEW

**Comment by DavidGizra** [comment]
Fix the topics row.

screenshot:
homepage_right

*in the issue you can see the error screenshot.

**Comment by DavidGizra** [comment]
Need to fix the footer.

footer

**Comment by DavidGizra** [comment]
In the image you can see that the footer jekyll fixed:

drupalfotter

File: phantomcss/homepage-drupal.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ phantomcss.init({
   libraryRoot: './node_modules/phantomcss'
 });
 
-casper.start('http://127.0.0.1:8080');
+casper.start('http://localhost/haskala/www/');
```
Comment (bitamar) [inline phantomcss/homepage-drupal.js]: Change back to 127.0.0.1:8080

File: haskala/modules/custom/haskala_terms/plugins/content_types/topics/topics.inc [hunk 2]
```diff
@@ -15,7 +15,7 @@ function haskala_terms_topics_content_type_render($subtype, $conf, $args, $conte
     ->fields('t', array('field_topic_tid'))
     ->groupBy('field_topic_tid');
   $query->addExpression('COUNT(field_topic_tid)', 'usage_count');
-  $query->having('usage_count > 8')->orderBy('usage_count', 'DESC');
+  $query->having('usage_count > 12')->orderBy('usage_count', 'DESC');
```
Comment (bitamar) [inline haskala/modules/custom/haskala_terms/plugins/content_types/topics/topics.inc]: Replcae this row with: 
```$query->orderBy('usage_count', 'DESC')->range(0, variable_get('haskala_topics_limit', 23));```

File: phantomcss/homepage-drupal.js (hunk 1 above)
Comment (amitaibu) [inline phantomcss/homepage-drupal.js]: (btw, this is something we'll need to figure how to determine the URL by the ``env`` - I tried it, but it still doesn't work)

File: behat/features/homepage-drupal.feature [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  @api @foo
+  Scenario Outline: Visit a homepage drupal page
+    Given I am an anonymous user
+    When  I visit ""
+    Then  I click "<link home page>"
+    And   I should see the link "<link>"
+
+  Examples:
+    | link home page | link |
+    | BOOKS          | A    |
```
Comment (bitamar) [inline behat/features/homepage-drupal.feature]: Change BOOKS to Books (The actual text is [or should] be uppercased with css)

Comment (DavidGizra) [inline behat/features/homepage-drupal.feature]: The test passes but the word has been changed

**Comment by DavidGizra** [comment]
@bitamar - Please check.

File: .travis.yml [hunk 4]
```diff
@@ -86,8 +86,9 @@ script:
 
 
   # Upload failed images to imgur
-#  - bash $TRAVIS_BUILD_DIR/imgur ./screenshots/drupal-homepage-tags-cloud-wrapper_3.png
-#  - bash $TRAVIS_BUILD_DIR/imgur ./failures/drupal-homepage-tags-cloud-wrapper_3.fail.png
+  - bash $TRAVIS_BUILD_DIR/imgur ./screenshots/drupal-homepage-tags-cloud-wrapper_5.png
```
Comment (bitamar) [inline .travis.yml]: Revert this

File: behat/features/homepage-drupal.feature [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Feature: HomePage Drupal
+  In order to be able to view a homepage drupal
+  As an anonymous user
+  We need to be able to have access to a homepage drupal page
+
+  @api @foo
+  Scenario Outline: Visit a homepage drupal page
+    Given I am an anonymous user
+    When  I visit ""
+    Then  I click "<link home page>"
```
Comment (bitamar) [inline behat/features/homepage-drupal.feature]: No need to click the homepage link, you're already on it

Comment (bitamar) [inline behat/features/homepage-drupal.feature]: Should probably be  ```Then I should see the link "<link>"```

File: phantomcss/homepage-drupal.js [hunk 6]
```diff
@@ -43,12 +52,6 @@ casper.then(function() {
     phantomcss.screenshot('.tags-cloud li a', 'drupal-homepage-tags-cloud-hover');
 });
 
-casper.then(function() {
```
Comment (bitamar) [inline phantomcss/homepage-drupal.js]: In the new images - I don't see an hovered "icons-nav", nor a topic link.

**Comment by bitamar** [comment]
@DavidGizra - Please make sure you check hover states

File: behat/features/homepage-drupal.feature (hunk 5 above)
Comment (DavidGizra) [inline behat/features/homepage-drupal.feature]: After then coming and :)
the Then is in the line before this.

File: phantomcss/homepage-drupal.js (hunk 6 above)
Comment (DavidGizra) [inline phantomcss/homepage-drupal.js]: I removed this test.

**Comment by bitamar** [comment]
@DavidKohav - Please export the failing images to imgur

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**